.github/workflows/release-packages.yaml

```diff
@@ -6,7 +6,7 @@ env:
 
 name: Upload additional release assets
 jobs:
-  ubuntu-22.04-package:
+  ubuntu-22_04-package:
```

Comment:
:pick: With regards to the commit title, this is the name of the job, not the package.